posts/2017-01-05-dat-next.md

```diff
@@ -1,9 +1,11 @@
 # Dat Next Preview Release
 
-Today we're announcing the release of Dat Next, our brand new Dat command line interface.
+Over the last few months we have been rebuilding the Dat command line tool and node modules to prepare for publishing datasets on our Dat registry. We've released a preview of this new command line tool you can install it now!
```

Comment:
What if we say 'we have been optimizing and improving the Dat command line tool as well as underlying dependencies, such as Hyperdrive' 

I think saying 'rebuilding' makes people nervous